templates/cli/lib/questions.js.twig

```diff
@@ -152,7 +153,9 @@ const questionsInitProject = [
         message: "Choose your organization",
         choices: async () => {
             let client = await sdkForConsole(true);
-            const { teams } = await paginate(teamsList, { parseOutput: false, sdk: client }, 100, 'teams');
+            const { teams } = client.endpoint === "https://cloud.appwrite.io/v1" 
```

Comment:
any chance we can check for *.appwrite.io? Since later on we're going to have multiple.